extensions/test/kokkos/kokkos_main.cpp

```diff
@@ -0,0 +1,55 @@
+// SPDX-FileCopyrightText: 2017-2023 The Ginkgo authors
```

Comment:
```suggestion
// SPDX-FileCopyrightText: 2017-2024 The Ginkgo authors
```
Also the other copyright notices needs to be updated